doc/markdownlinkconverter/markdownlinkconverter.c

```diff
@@ -0,0 +1,452 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <ctype.h>
+#include <stdbool.h>
+#include <string.h>
+
+#if defined(WIN32)
+#define FOLDER_DELIMITER '\\'
+#else
+#define FOLDER_DELIMITER '/'
+#endif
+
+/* Slashes or backslashes to the elektra root.
+ * Depends on the place of the executable, example:
+ * /foo/bar/elektra/build/bin/executable
+ * E:\\foo\bar\elektra\build\bin\executable
+ * In both cases the there are 3 slashes or backslashes
+ */
+#define FOLDER_DELIMITER_TO_ELEKTRA_ROOT 3
+#define TEMP_FILENAME "temp"
+
+// Link Blacklist: do not convert links with the following starting and ending
+char * ignoreTargetEnd [] = { ".h", ".c" , ".cpp" , ".hpp" , "\0"};
+// ignore http, @ref and #anchor
+char * ignoreTargetStart [] = { "#", "@" , "http" , "\0"};
+// both need to be null terminated
```

Comment:
Do you mean null termination in addition to the element `"\0"` that contains two null bytes?